maps-compose-widgets/build.gradle

```diff
@@ -0,0 +1,51 @@
+plugins {
+    id 'kotlin-android'
+    id 'kotlin-parcelize'
+}
+
+android {
+    compileSdk 31
+
+    defaultConfig {
+        minSdk 21
+        targetSdk 31
+        versionCode 1
+        versionName "1.0"
+    }
+
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_8
+        targetCompatibility JavaVersion.VERSION_1_8
+    }
+
+    composeOptions {
+        kotlinCompilerExtensionVersion "$compose_version"
+    }
+
+    buildFeatures {
+        buildConfig false
+        compose true
+    }
+
+    kotlinOptions {
+        jvmTarget = '1.8'
+        freeCompilerArgs += '-Xexplicit-api=strict'
+        freeCompilerArgs += '-Xopt-in=kotlin.RequiresOptIn'
+    }
+}
+
+dependencies {
+    implementation "androidx.compose.foundation:foundation:$compose_version"
+    implementation 'com.google.maps.android:maps-compose:2.2.1'
+    implementation 'androidx.compose.material:material:1.1.1'
+    implementation 'androidx.core:core-ktx:1.7.0'
+    implementation 'com.google.android.gms:play-services-maps:18.0.2'
+    implementation 'com.google.maps.android:maps-ktx:3.4.0'
+    implementation 'com.google.maps.android:maps-utils-ktx:3.3.0'
```

Comment:
suggestion: dependencies shared across modules should be defined in a separate gradle file so there is one version used by both artifacts. doesn't have to be addressed now but something for future reference.

Reply:
Sounds good - I opened https://github.com/googlemaps/android-maps-compose/issues/160 for this

Reply:
Sounds good. I opened https://github.com/googlemaps/android-maps-compose/issues/160 for this.